website/content/docs/k8s/installation/vault/gossip.mdx

```diff
@@ -18,36 +18,44 @@ To use a gossip encryption key stored in Vault we need the following:
 First, generate and store the gossip key in Vault:
 
 ```shell-session
-vault kv put secret/consul/gossip key="$(consul keygen)"
+$ vault kv put secret/consul/gossip key="$(consul keygen)"
 ```
 
 Next, we will need to create a policy that allows read access to this secret:
 
-```shell-session
-# gossip-policy.hcl
+
+<CodeTabs>
+
+<CodeBlockConfig filename="gossip-policy.hcl">
+
+```HCL
```

Comment:
@david-yu we only use code tabs when we are showcasing two configurations in a different form. Such as HCL/JSON. If you want to display the file name you can keep the `CodeBlockConfig` with the filename. Otherwise, the code tabs don't make sense.